### ISSUE TITLE
Crash in VungleBannerCustomEvent.m for iOS 9

### ISSUE THREAD

**Issue by dandepeched** [mopub/mopub-ios-mediation#279]
1. **What is the issue?**
    Application is crashing during MoPub initialization on iOS 9 devices. `scheduledTimerWithTimeInterval:repeats:block:` method is supported starting from iOS10 and should be wrapped with `if (@available(iOS 10.0, *))`:
```
Fatal Exception: NSInvalidArgumentException
+[NSTimer scheduledTimerWithTimeInterval:repeats:block:]: unrecognized selector sent to class 0x37c30ed4
-[VungleBannerCustomEvent requestAdWithSize:customEventInfo:adMarkup:]
```

2. **Check the applicable box(es) below by placing an `x` in the brackets:**
    - [x] I searched https://developers.mopub.com/ and found no answer.
    - [x] I searched https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed.
    - [x] I searched the adapter changelogs to make sure this issue has not already been resolved.

3. **Fill in the below template as detailed as possible:**
    * **Ad Format** (*Banner, Interstitial, Rewarded Video, Native Static, Native Video*)
        > Banner
    * **Platform** (*Android, iOS, Unity*)
        > iOS
    * **MoPub SDK Version**
        > 5.12.1
    * **Mediation SDK Name & Version**
        > Vungle 6.5.3
    * **Mediation Adapter Version**
        > 6.5.3
    * **MoPub Ad Unit ID(s) used to reproduce the issue**
        > Issue not related to Ad Unit placement
    * **Device model and OS Version**
        > Any device with iOS9
    * **Steps to reproduce the issue** (include integration steps if not standard)
        > 1. Integrate Vungle mediation
        > 2. Start application on iOS9 device
    * **Observed behavior**
        > Crash on iOS9 in VungleBannerCustomEvent.m
    * **Expected behavior** (if known)
        > iOS9 should be supported (because MoPub SDK supports it)
    * **Evidence** (if available)
        > [Fatal Exception NSInvalidArgumentException.txt](https://github.com/mopub/mopub-ios-mediation/files/4535255/Fatal.Exception.NSInvalidArgumentException.txt)

**Comment by dravyan** [comment]
@dandepeched, thank you for reporting this. We will update the adapters to resolve this crash. I will notify this thread once the updated adapters are released.

Thanks,
MoPub Team

**Comment by dravyan** [comment]
@dandepeched , this issue has been fixed in the latest adapters `[6.5.3.1](https://github.com/mopub/ios-mediation/blob/master/Vungle/CHANGELOG.md#changelog)`. Please update.

Thanks,
MoPub Team
